third_party/conan/docker/digests.sh

```diff
@@ -16,20 +16,21 @@ declare -rA docker_image_digest_mapping=(
   [clang9_release]="gcr.io/orbitprofiler/clang9_release@sha256:02f1a69425bcf6039e309da82c1bcf4aa2aabbe1d45e4b481579b08b76f2a29d" \
   [clang9_relwithdebinfo]="gcr.io/orbitprofiler/clang9_release@sha256:02f1a69425bcf6039e309da82c1bcf4aa2aabbe1d45e4b481579b08b76f2a29d" \
   [clang9_debug]="gcr.io/orbitprofiler/clang9_debug@sha256:02f1a69425bcf6039e309da82c1bcf4aa2aabbe1d45e4b481579b08b76f2a29d" \
-  [clang11_release]="gcr.io/orbitprofiler/clang11_release@sha256:adc87a0017c650e08bb27748d22db373fce48bea3b35f5fd5188e46c161e74b3" \
-  [clang11_relwithdebinfo]="gcr.io/orbitprofiler/clang11_release@sha256:adc87a0017c650e08bb27748d22db373fce48bea3b35f5fd5188e46c161e74b3" \
-  [clang11_debug]="gcr.io/orbitprofiler/clang11_debug@sha256:adc87a0017c650e08bb27748d22db373fce48bea3b35f5fd5188e46c161e74b3" \
+  [clang11_release]="gcr.io/orbitprofiler/clang11_debug@sha256:161a84fc066a6c6182a2c89b1f2eeff3354aa383ebc270dce600d4943f2880dc" \
+  [clang11_relwithdebinfo]="gcr.io/orbitprofiler/clang11_debug@sha256:161a84fc066a6c6182a2c89b1f2eeff3354aa383ebc270dce600d4943f2880dc" \
```

Comment:
I don't know what all of this is for, but looking at other lines, why `clang11_debug` instead of `clang11_release`?

Reply:
All clang profiles use the same container, that's why the reference is all the same. It's debug because it's alphabetically the first profile name.